reservation/test_appointment_model.py

```diff
@@ -0,0 +1,69 @@
+from reservation.models import Appointment, TypeOfJob
+from datetime import datetime
+import pytest
+
+START_APPOINTMENT = datetime(2023, 4, 17, 12, 0, 0)
+END_APPOINTMENT = datetime(2023, 4, 17, 13, 0, 0)
+SUMMARY = ""
+
+
+@pytest.fixture
+def persisted_appointment(appointment):
+    appointment.typeOfJob_id.save()
+    appointment.professional_id.save()
+    appointment.client_id.save()
+    appointment.save()
+    return appointment
+
+
+@pytest.fixture
+def persisted_appointment_pool(persisted_appointment, typeOfJob, professional, client2):
```

Comment:
Please use this instead of calling for fixture client/pro/profile that calls the same with #numbers

Review this 
https://www.inspiredpython.com/article/five-advanced-pytest-fixture-patterns

Reply:
Can you explain to me the problem with what I did? And besides, I didn't quite understand the solution you offer

Reply:
OK, instead of calling two fixtures create_client1 and create_client2
You can have create_client and call it with parameters inside the test.
this way you have 2 clients or proffesional with one fixture

The same work for pools of appointments and so on.
please review and understand the concept again

Reply:
I understand what you said but the changes you suggested should be in conftest.py? Won't this cause tests to fail? Because everyone will have to change their tests after my changes

Reply:
I meant to the appointments
Who ever will fix the client, will fix the other failures.

So your appointment pool can work with client and client1
Please refer the slavck article parameterized and factories, in slack

Reply:
i have in my PR the Factory way for the account members, https://github.com/redhat-beyond/Pro4U/pull/75